README.md

```diff
@@ -6,10 +6,10 @@ This repository contains a set of scripts for creating, maintaining, and releasi
 ## Python Script
 A front-end script, git_vivado.py, is provided to parse command line arguments and call into Vivado at the command line.  This script has three subcommands: "checkout", "checkin", and "release". Each of these subcommands has its own help menu, which explains the arguments that can be passed to the script, as well as show what the default values of each of these arguments will be. All paths passed to the script are assumed to be relative to the current working directory.
 ### Checkout
-This subcommand calls into digilent_vivado_checkout.tcl in order to create a Vivado project, in the form of an XPR file, using the sources and scripts contained in the project repository. It takes up to three additional arguments: a path to the XPR file to be created ('-x'), a path to the repository to use to create the XPR ('-r'), and the version number of the Vivado installation to be used ('-v'). Use the "help" interface by calling "python git_vivado.py checkout -h" for more information and to view the script's default arguments.
```

Comment:
in case you can't see the difference, like me, in these 540 characters I only changed python to python3.